src/vs/workbench/contrib/interactive/browser/interactive.contribution.ts

```diff
@@ -284,20 +293,24 @@ registerAction2(class extends Action2 {
 		});
 	}
 
-	async run(accessor: ServicesAccessor, column?: number, resource?: URI, id?: string): Promise<{ notebookUri: URI, inputUri: URI; }> {
+	async run(accessor: ServicesAccessor, showOptions?: { viewColumn?: number, preserveFocus?: boolean }, resource?: URI, id?: string): Promise<{ notebookUri: URI, inputUri: URI; }> {
```

Comment:
to ensure the jupyter extension is not broken when this change is shipped, the code here can be backwards compatible, like `showOptions?: number | { viewColumn?: number, preserveFocus?: boolean }`, and then remove it once Jupyter releases.